README.md

```diff
@@ -33,7 +33,7 @@ To install in a given project, you can add `nvd-clojure/nvd-clojure {:mvn/versio
 If you have CLI version 1.10.3.933 or later, you can also install `nvd-clojure` as a "tool":
 
 ```bash
-clojure -Ttools install nvd-clojure/nvd-clojure '{:mvn/version "RELEASE"}' :as nvd
+clojure -Ttools install io.github.rm-hull/nvd-clojure '{:git/tag "1.8.0"}' :as nvd
```

Comment:
I'd prefer this was a Maven version, since it's the one thing that we always release (today I released 1.8.0 and forgot to tag for example*)

*will add it to the Makefile that performs deploys. Then again, people can deploy things by hand.